distributed/cli/dask_scheduler.py

```diff
@@ -59,8 +59,9 @@
 @click.option('--local-directory', default='', type=str,
               help="Directory to place scheduler files")
 @click.option('--preload', type=str, multiple=True, is_eager=True,
-              help='Module that should be loaded by each worker process '
-                   'like "foo.bar" or "/path/to/foo.py"')
+              help='Module that should be loaded by the scheduler process  '
+                   'like "foo.bar" or "/path/to/foo.py". Scheduler plugins '
+                   'should be loaded here.')
```

Comment:
Thanks for changing worker -> scheduler here. 

I'm not convinced that we should call out the specific use case of scheduler plugins though.  Preload scripts are used for a variety of reasons, of which this is only one.

Reply:
Should we remove this?

Reply:
Yeah good point, removed.